app/uninstall.go

```diff
@@ -15,6 +12,9 @@ import (
 	"github.com/longhorn/longhorn-manager/datastore"
 	lhclientset "github.com/longhorn/longhorn-manager/k8s/pkg/client/clientset/versioned"
 	lhinformers "github.com/longhorn/longhorn-manager/k8s/pkg/client/informers/externalversions"
+	"github.com/pkg/errors"
```

Comment:
Don't move those to longhorn specific section.

Reply:
Fixed. @yasker